### PR TITLE
Feature/admin review crud

### PR DESCRIPTION
**Pull request by Juanda0**
In this PR I include
- The Creation of the view for the review CRUD
- The DB migrations 
- The DB seeding

### CODE REVIEW

File: app/Http/Controllers/ReviewController.php [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    {
+        $viewData = []; //to be sent to the view
+        $viewData['title'] = __('admin.createReview');
+
+        return view('admin.review.create')->with('viewData', $viewData);
+    }
+
+    public function save(Request $request): RedirectResponse
+    {
+        $request->validate([
```
Comment (Jguerra47) [inline app/Http/Controllers/ReviewController.php:48]: You should not validate the request to create the review in the controller. You could create a function from the Review model.
```suggestion
        Review::validate(request);
```

File: database/seeders/DatabaseSeeder.php [hunk 2]
```diff
@@ -13,7 +14,7 @@ class DatabaseSeeder extends Seeder
     public function run(): void
     {
         // \App\Models\User::factory(10)->create();
```
Comment (Jguerra47) [inline database/seeders/DatabaseSeeder.php:16]: You could remove this comments.

**Review by Jguerra47 — CHANGES_REQUESTED** [review]
Some checks, but good work! 🚀

File: app/Http/Controllers/ReviewController.php (hunk 1 above)
Comment (Juanda0) [reply app/Http/Controllers/ReviewController.php:48]: Fixed

File: database/seeders/DatabaseSeeder.php (hunk 2 above)
Comment (Juanda0) [reply database/seeders/DatabaseSeeder.php:16]: Fixed

**Review by Jguerra47 — APPROVED** [review]
LGTM

**Pull request merged**